docs/developer-docs/multi-chain/overview.mdx

```diff
@@ -0,0 +1,115 @@
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Advanced", "Multi-chain", "Concept" ]} />
+
+## What is multi-chain?
+
+The term 'multi-chain' refers to the Internet Computer's ability to trustlessly make direct calls to another blockchain, enabling dapps to expand their capabilities across multiple networks. ICP smart contracts can interact with another chain's smart contracts and process the chain's native assets. This ability to communicate directly with other blockchain networks is possible using the protocol's unique cryptography features, which support signing transactions that execute on other networks. ICP currently has multi-chain integrations with the Bitcoin and Ethereum networks.
+
+Traditionally, for one blockchain to communicate in this manner with other chains, they need to use a blockchain 'bridge'. Traditional bridges introduce several problems:
+
+- The bridge is an intermediary service that must be trusted to handle transactions and assets honestly and properly.
+
+- Bridges can be subjects of security vulnerabilities and exploits.
+
+- Transaction times have a high amount of latency.
+
+- Bridges do not meet regulatory standards.
+
+On ICP, there is no bridge, hub, or other intermediary — just two chains communicating directly.
+
+## How multi-chain on ICP works
+
+Multi-chain is possible through the unique features of ICP: [threshold ECDSA signatures](/docs/current/references/t-ecdsa-how-it-works), chain-key cryptography, and [HTTPS outcalls](/docs/current/references/https-outcalls-how-it-works).
+
+Threshold ECDSA signatures, combined with chain-key cryptography, allow ICP smart contracts to create [signed transactions](/docs/current/developer-docs/smart-contracts/encryption/signing-transactions) on other networks. By submitting transactions directly to other chains, ICP dapps can interact directly with another chain's smart contracts, assets, and tokens.
+
+HTTPS outcalls are used to communicate with external entities, and are used by the EVM RPC canister, allowing smart contracts to interact with JSON-RPC services for EVM-compatible chains.
+
+Integrations with different chains vary in how they utilize these features to facilitate the integration.
+
+### Bitcoin integration
+
+The integration with Bitcoin primarily uses the threshold ECDSA feature along with a Bitcoin adapter. Threshold ECDSA is used to securely store a smart contract's secret key, which is used to sign transactions. Bitcoin supports ECDSA, allowing ICP smart contracts to directly create and sign transactions on the Bitcoin network. Canisters can also create a Bitcoin address, then send and receive bitcoin as if they were a traditional Bitcoin user.
+
+The Bitcoin adapter is a subnet of nodes that participate in the Bitcoin network. They fetch the Bitcoin network state and blocks, then facilitate sending transactions to the Bitcoin network.
```

Comment:
> The Bitcoin adapter is a subnet of nodes that participate in the Bitcoin network. 

I don't think this is accurate. The Bitcoin adapter is a process that runs side by side with the replica on IC nodes (technically on all nodes but only active on the nodes of the subnet where the Bitcoin canisters live) and communicates with the Bitcoin network. They don't really participate in the sense that they don't mine blocks -- they simply follow the bitcoin chain (you can loosely consider the bitcoin adapter our own custom lightweight bitcoin client).

Reply:
Agreed. I am not even sure it these technical details need to be on the overview page?

I would write something vague like, the IC nodes have a special extension called the Bitcoin adapter allowing them to _directly_ fetch the Bitcoin network state and blocks, then facilitate sending transactions to the Bitcoin network.

Reply:
Revised to "ICP nodes have a special extension called the Bitcoin adapter allowing them to directly fetch the Bitcoin network state and blocks, then facilitate sending transactions to the Bitcoin network."